### ISSUE TITLE
:beetle: not rendering due to bad raw

### ISSUE THREAD

**Issue by timheuer** [hlaueriksson/GEmojiSharp#6]
The `:beetle:` is not rendering correctly

**Comment by hlaueriksson** [comment]
Hi @timheuer 

I think that the current raw `:beetle:` emoji is correct, even though it looks broken on some systems.
It is taken from https://github.com/github/gemoji/blob/master/db/emoji.json#L7899
And it is the same as https://emojipedia.org/beetle/

It looks broken on Windows 10 and some other operating system I tested with.
On Android 11, it looks good to me:

![Screenshot_20201121-140729](https://user-images.githubusercontent.com/1299786/99878212-0b89be80-2c04-11eb-82e2-0bcf83c724a9.jpg)

**Comment by hlaueriksson** [comment]
However, it looks like GitHub mixed up the rendering of `:beetle:` and `:lady_beetle:`

https://github.com/hlaueriksson/github-emoji#user-content-beetle

![image](https://user-images.githubusercontent.com/1299786/99879611-5b6d8300-2c0e-11eb-8309-51367aeac77b.png)

`:lady_beetle:` should look like https://emojipedia.org/lady-beetle/

**Comment by hlaueriksson** [comment]
@timheuer do you think posting an issue about this in https://github.com/github/g-emoji-element is the the right way forward?

**Comment by timheuer** [comment]
If you look at https://emojipedia.org/lady-beetle/ definition and the Shortcodes section it indicates
```
:beetle: (Github, Slack)
```

**Comment by hlaueriksson** [comment]
I found issues with two other emojis:

```
:man_in_tuxedo: 🤵‍♂️ != 🤵
:bride_with_veil: 👰‍♀️ != 👰
```

**Comment by hlaueriksson** [comment]
The `:beetle:` emoji is fixed in version `1.4.0` of `GEmojiSharp`:
https://www.nuget.org/packages/GEmojiSharp/1.4.0

@timheuer Thanks for the pull request!

**Comment by hlaueriksson** [comment]
Hi again @timheuer 

It looks like GitHub changed the rendering of the `:beetle:` emoji.

I wanted to give you a heads up that on the upcoming version `1.5.0` of `GEmojiSharp`, you need to use the `:lady_beetle:` emoji to render a :lady_beetle:.
